features/fields.feature

```diff
@@ -171,84 +208,89 @@ Functionaliteit: Fields
       | naam.aanduidingNaamgebruik.code |
       | aanduidingNaamgebruik.code      |
 
+    Abstract Scenario: Zoek met geslachtsnaam en geboortedatum: opgeven <pad> levert het veld
+      Als personen wordt gezocht met de volgende parameters
+      | naam          | waarde                              |
+      | type          | ZoekMetGeslachtsnaamEnGeboortedatum |
+      | geslachtsnaam | Kierkegaard                         |
+      | geboortedatum | 1956-11-15                          |
+      | fields        | burgerservicenummer,<pad>           |
+      Dan heeft de response een persoon met alleen de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999991929 |
+      En heeft de persoon alleen de volgende 'geboorte' gegevens
+      | naam              | waarde           |
+      | datum.type        | Datum            |
+      | datum.datum       | 1956-11-15       |
+      | datum.langFormaat | 15 november 1956 |
+
+
+      Voorbeelden:
+      | pad                  |
+      | geboorte.datum.datum |
+      | datum.datum          |
+
     Abstract Scenario: opgegeven pad is volledig pad én deel van pad
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
       | burgerservicenummer | 999991929                       |
       | fields              | burgerservicenummer,<pad>       |
-      Dan heeft de persoon met burgerservicenummer '999991929' alleen de volgende gegevens
+      Dan heeft de response een persoon met alleen de volgende gegevens
       | naam                | waarde    |
       | burgerservicenummer | 999991929 |
       | <pad>               | <waarde>  |
 
       Voorbeelden:
-      | pad                   | waarde      | opmerking                                                         |
-      | geslacht.omschrijving | vrouw       | levert geslacht.omschrijving, niet partners.geslacht.omschrijving |
-      | naam.voornamen        | Kierkegaard | levert naam.voornamen, niet partners.naam.voornamen               |
+      | pad                   | waarde         | opmerking                                                         |
+      | geslacht.omschrijving | vrouw          | levert geslacht.omschrijving, niet partners.geslacht.omschrijving |
+      | naam.voornamen        | Yvonne Johanna | levert naam.voornamen, niet partners.naam.voornamen               |
 
   Rule: Wanneer velden van polymorfe gegevensgroep wordt gevraagd, wordt altijd het 'type' veld van de gegevensgroep terug gegeven
     - wanneer het gegeven geen waarde heeft (ook niet onbekend) wordt de gegevensgroep niet geleverd en dus ook 'type' niet teruggegeven

```

Comment:
deze sub-regel geldt niet meer door onze beslissing over het leveren van een leeg object. Onderstaand scenario doet dat al goed, maar komt niet overeen met wat de regel beschrijft. Volgens die sub-regel heeft het gevraagde gegeven geen waarde en moet dus ook type niet worden teruggegeven.

Los je dat nu op of met #1076?
Dus deze regel "wanneer het gegeven geen waarde heeft..." moet denk ik worden vervangen door twee regels:
- niet leveren leeg object wanneer het object niet bestaat: geen enkel gegeven, gevraagd of niet-gevraagd, heeft een waarde
- wel leveren leeg object wanneer object wel bestaat maar gevraagd veld heeft geen of lege waarde: een onbekendwaarde (bijv. 0000 voor land of "." voor geslachtsnaam) in een gegeven geldt in dit geval als wel bestaan van het object.

Reply:
Ik haal de sub-regel weg en los het op met #1076